src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const population = document.getElementsByClassName('population');
+const populationArray = [...population]
+  .map(el => +el.textContent.split(',').join(''));
+
+const total = populationArray.reduce((sum, el) => sum + el, 0);
+const average = total / populationArray.length;
+const totalSpan = document.querySelector('.total-population');
+const averageSpan = document.querySelector('.average-population');
+
+totalSpan.textContent = total.toLocaleString();
+averageSpan.textContent = average.toLocaleString();
```

Comment:
You should return the number in the format 452,543,964 instead of 452 543 964.
Try to change locale to 'en-US'

Reply:
Fixed, thanks!